src/checks/pokerus.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  description: 'Illegal pokerus strain for Pokémon',
+  field: 'pokerus',
+  test (pkmn) {
+    const strain = pkmn.pokerusStrain;
+    const duration = pkmn.pokerusDuration;
+
+    if (strain < 0 || strain > 15 || duration < 0 || duration > 4) {
+      return false;
+    }
+    if (strain === 0) {
+      return !(strain === 0 && duration > 0);
+    }
+    if (strain >= 0 || strain <= 15) {
+      const maxDays = strain % 4 + 1;
+      return maxDays > duration;
```

Comment:
If I'm reading this correctly, `maxDays` will never be more than 4 here. So the check on line 8 is actually redundant, because if `duration` is greater than 4, then `maxDays > duration` will always be `false`.